packages/helper/src/main/ts/utils/converters.ts

```diff
@@ -0,0 +1,16 @@
+import { IComplexDelay, ILimit, ILimitStack } from 'push-it-to-the-limit/target/es5/interface'
+
+import { TRateLimitOpts } from '../helper'
+
+const complexDelayToLimit = (delay: IComplexDelay): ILimit => ({
```

Comment:
This utils should be exported and used instead:

https://github.com/antongolub/push-it-to-the-limit/blob/d1b4360935af28313ec6be4e303b1b220a9e5729/src/main/ts/common.ts#L79

Reply:
but we need to convert IComplexDelay to ILimit, normalizeDelay just returns array of IComplexDelay, which is not accepted by rateLimit